testing/test_pytester.py

```diff
@@ -744,10 +744,14 @@ def test_run_result_repr() -> None:
 
     # known exit code
     r = pytester_mod.RunResult(1, outlines, errlines, duration=0.5)
-    assert (
-        repr(r) == "<RunResult ret=ExitCode.TESTS_FAILED len(stdout.lines)=3"
-        " len(stderr.lines)=4 duration=0.50s>"
-    )
+    expected = [
```

Comment:
I suggest we just make it consistent here -- make it use one or the other (the new one looks better to me)

Reply:
I suppose we definitely will search for "3.10" when dropping that version, so in the end doesn't really matter I guess.